docs/guide/worker.md

```diff
@@ -46,9 +46,9 @@ Worker starting in daemon mode with `queue/listen` command supports [File], [Db]
 Systemd
 -------
 
-Systemd is an init system used in Linux to bootstrap the user space. To configure a start of workers
-using systemd, create config file named `yii-queue@.service` in `/etc/systemd/system`, that
-will contain:
+Systemd is a Linux system manager used to bootstrap daemons. To configure workers startup
```

Comment:
this was correct before, now its not really true. Systemd is an init system, not a manager for daemons.